code-challenges.js

```diff
@@ -24,22 +24,48 @@ const fruit2 = "banana"
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const aaa = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+// console.log(aaa.length)
+
 // Expected output: 9
 
 // Pseudo code:
+//For line 29 im setting a varible that combines the array of lines 27 and 28 together using .concat
+//Line 30. Since im combined my arrays together. I can call upon the set variable and take its total length = 9.
```

Comment:
It's best practice to make your variable names relate to the data it's holding. You don't want to be working on a big project with 100 variables each named "aaa" "aaaa" "aaaaa" so on and so forth.